features/unattached_status.feature

```diff
@@ -599,3 +599,26 @@ Feature: Unattached status
         Examples: ubuntu release
            | release | machine_type  |
            | jammy   | lxd-container |
+
+    Scenario Outline: Check notice file read permission
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I run `mkdir -p /run/ubuntu-advantage/notices` with sudo
+        When I run `touch /run/ubuntu-advantage/notices/crasher` with sudo
+        When I run `chmod 0 /run/ubuntu-advantage/notices/crasher` with sudo
+        When I run `mkdir -p /var/lib/ubuntu-advantage/notices` with sudo
+        When I run `touch /var/lib/ubuntu-advantage/notices/crasher` with sudo
+        When I run `chmod 0 /var/lib/ubuntu-advantage/notices/crasher` with sudo
+        When I run `touch /var/lib/ubuntu-advantage/notices/10-reboot_required` with sudo
```

Comment:
Now that we're checking for valid notices, this one doesn't show.
This notice is a non-permanent one
```suggestion
        When I run `touch /run/ubuntu-advantage/notices/10-reboot_required` with sudo
```